src/Effects.js

```diff
@@ -654,7 +654,6 @@ module.exports = class Effects {
           p5.rotate(poop.rot);
           p5.scale(2, 2);
           drawFallingPoop(p5._renderer.drawingContext);
-          p5.pop();
```

Comment:
@simplycloud It looks like there were two calls to `p5.pop`. I removed one, but want to double check with you that this won't break expected outcomes.